scc/committee.go

```diff
@@ -35,13 +35,20 @@ type MemberId uint16
 // NewCommittee creates a new committee with the provided members. There must be
 // at least one member, all members must be valid, and no duplicates are allowed.
 func NewCommittee(members ...Member) (Committee, error) {
-	res := Committee{members: slices.Clone(members)}
+	res := NewCommitteeForTests(members...)
```

Comment:
based on the documentation for `NewCommitteeForTests`, it should not be used for production code, which `NewCommittee` looks like, right?